intro_to_r/intro_to_r.Rmd

```diff
@@ -39,17 +39,33 @@ To get you started, enter the following commands at the R prompt (i.e. right aft
 `>` on the console).  You can either type it in manually or copy and paste it
 from this document.
 
-```{r load-abrbuthnot-data, eval=TRUE, message = FALSE}
+R has a number of additional packages that help you perform specific tasks. In order to use these packages, they must be installed (you only have to do this once) and loaded (you have to do this every time you start an R session). 
+
+```{r opts, eval=TRUE, echo=FALSE}
+library(knitr)
+opts_chunk$set(eval=FALSE)
+```
+
+If you have not installed the packages already, run these commands:
+```{r load-devtools-oilabs, message = FALSE, eval=FALSE}
 devtools::install_github("andrewpbray/oilabs")
+install.packages(c("dplyr", "ggplot2"))
```

Comment:
You could simplify this a bit by making dplyr and ggplot2 dependencies (even if slightly spurious) of the oilabs package.